source/mainnet/smart-contracts/tutorials/counter/counter-contract.rst

```diff
@@ -25,215 +25,211 @@ Once you have set up the tools needed you are ready to create your smart contrac
 
     $cargo concordium init
 
-Select the ``default`` option from the menu.
+When prompted for which template to expand, select the ``default`` option. You will then be asked for a name for your project. In this example we'll use "counter", but you can choose whatever name you want.
 
-.. image:: images/select-default.png
-    :width: 100%
+The result is a basic skeleton of a smart contract. Initially, it has a ``State`` struct, an ``init`` function for creating new instances, an ``Error`` enum for custom errors, a ``view`` function to read the state, and a dummy ``receive`` function.
 
-Then it will ask for a name and a description of your project. Fill them in.
-The result is a basic skeleton of a smart contract.
-Initially, it has a ``State`` struct, an ``init`` function for creating new instances, an ``Error`` enum for custom errors, a ``view`` function, and a ``receive`` function.
-
-.. image:: images/contract.png
```

Comment:
I disagree - I don't think the screenshots are very helpful, even for beginners. Even if they help a little, they are much more likely to get out of date than the rest of the tutorial, so I'd rather not include them, as they'll likely soon be more confusing than helpful.

>developer audience who can barely code

Is this reasonable? I don't think it's realistic to expect anyone "who can barely code" to get through these tutorials with much understanding. Have we actually tested the tutorial on such people? If you ask me, this tutorial already assumes plenty of stuff that would make it totally infeasible for anyone on a lower level than a hobby developer.

Reply:
I think it would be useful to see the output of the `concordium-client` at least once for the basic commands like `deploy/init/update` (these commands I expect not to change anytime soon). I leave it to you to decide if that adds value.

Reply:
I can potentially add the output as text in the console code block, instead of as a screenshot - easier to update as well.